### ISSUE TITLE
Feature request / search: Trace log

### ISSUE THREAD

**Issue by leocasarsa** [probcomp/bdbcontrib#131]
Is there any way to list all the - mutation - BQL commands that have been executed in a specific bdb file? I don't need to know about ```'''SELECT * FROM %t'''```, but I would like to know things like ```'''CREATE TABLE foo'''``` or ```ANALYZE 24 MODELS FOR foo```.

**Comment by riastradh-probcomp** [comment]
Can't quite report `ANALYZE`, but you can get an approximation with:

    echo .dump | sqlite3 ./foo.bdb

**Comment by BelhalK** [comment]
```
q('''SELECT iterations FROM bayesdb_generator_model
                           WHERE generator_id = (
                            SELECT id FROM bayesdb_generator WHERE name = %g)''')
```

is what you want I believe for `ANALYZE`

for `CREATE` your request #129 is relevant

**Comment by leocasarsa** [comment]
Thanks, both of you. Maybe we can integrate these two answers into a bdb.trace_log() method?

**Comment by gregory-marton** [comment]
I agree that we should be able to replay various views of a user's history to them to some extent. This is one way that session capture could be made to directly help users. One view should absolutely be times that tables got created or modified, and indeed how generators have been created and modified. Good idea!
